reports/202101_apache_report.md

```diff
@@ -0,0 +1,52 @@
+[DRAFT][REPORT] Apache Accumulo - January 2021
+
+## Description:
+
+The Apache Accumulo sorted, distributed key/value store is a robust, scalable,
+high performance data storage system that features cell-based access control
+and customizable server-side processing. It is based on Google's BigTable
+design and is built on top of Apache Hadoop, Zookeeper, and Thrift.
+
+## Issues:
+
+No change since last report, Oct 2020.  Still waiting for the owner of 
```

Comment:
Maybe add a little context RE why they are unable to at this time? That might help avoid the concern that they are avoiding helping transition ownership.

Reply:
updated with more current infomation